specification/app/resource-manager/Microsoft.App/stable/2025-01-01/examples/SessionPools_CreateOrUpdate.json

```diff
@@ -0,0 +1,193 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "rg",
+    "sessionPoolName": "testsessionpool",
+    "api-version": "2025-01-01",
+    "sessionPoolEnvelope": {
+      "location": "East US",
+      "identity": {
+        "type": "SystemAssigned"
+      },
+      "properties": {
+        "environmentId": "/subscriptions/34adfa4f-cedf-4dc0-ba29-b6d1a69ab345/resourceGroups/rg/providers/Microsoft.App/managedEnvironments/demokube",
+        "poolManagementType": "Dynamic",
+        "containerType": "CustomContainer",
+        "scaleConfiguration": {
+          "maxConcurrentSessions": 500,
+          "readySessionInstances": 100
+        },
+        "dynamicPoolConfiguration": {
+          "lifecycleConfiguration": {
+            "type": "OnContainerExit",
```

Comment:
shall we use `Timed` as sample given it is more commonly used, or we create 2 samples, one for `Timed` and one for `OnContainerExit`

Reply:
Yes, thanks, another sample added and there is one for `Timed` and one for `OnContainerExit` now